apps/blog/_posts/think-before-you-code.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Por trás de um grande programa há um grande plano"
+date: "2023-04-15"
+author:
+  name: "Miguel Ângelo Freitas"
+  photo: "mikefreitas.jpg"
+  username: "MrNameless10"
+draft: false
+topic: "Programação"
+featured: false
+---
+
+---
```

Comment:
```suggestion
```

Reply:
I think the reason why the CI is failing might be due to this particular line :sweat_smile: